projects/1.18/assets/delightful/delightful/lang/zh_cn.json

```
@@ -0,0 +1,90 @@
+  "item.delightful.green_tea_leaf": "绿茶叶",
+  "item.delightful.salmonberries": "大树莓",
+  "item.delightful.salmonberry_pips": "大树莓籽",
+  "item.delightful.matcha": "抹茶",
+  "item.delightful.acorn": "橡子",
+  "item.delightful.animal_fat": "动物脂肪",
+  "item.delightful.animal_oil_bottle": "荤油瓶",
+  "item.delightful.chopped_clover": "切碎的三叶草",
+  "item.delightful.cactus_flesh": "仙人掌肉",
+  "item.delightful.cactus_steak": "仙人掌肉排",
+  "item.delightful.field_salad": "田园沙拉",
+  "item.delightful.rock_candy": "石头糖",
+  "item.delightful.venison_chops": "生鹿排",
+  "item.delightful.cooked_venison_chops": "熟鹿排",
+  "item.delightful.raw_goat": "生山羊肉",
+  "item.delightful.cooked_goat": "熟山羊肉",
+  "item.delightful.bone_knife": "骨刀",
+  "item.delightful.amethyst_knife": "紫水晶刀",
+  "item.delightful.emerald_knife": "绿宝石刀",
+  "item.delightful.copper_knife": "铜刀",
+  "item.delightful.tin_knife": "锡刀",
+  "item.delightful.steel_knife": "钢刀",
+  "item.delightful.enderite_knife": "末影合金刀",
+  "item.delightful.obsidian_infused_enderite_knife": "黑曜石灌注末影合金刀",
+  "item.delightful.bronze_knife": "青铜刀",
+  "item.delightful.lapis_lazuli_knife": "青金石刀",
+  "item.delightful.osmium_knife": "锇刀",
+  "item.delightful.refined_glowstone_knife": "强化荧石刀",
+  "item.delightful.refined_obsidian_knife": "强化黑曜石刀",
+  "item.delightful.large_amethyst_knife": "紫晶刀",
+  "item.delightful.black_opal_knife": "黑欧泊刀",
+  "item.delightful.netherite_opal_knife": "下界合金欧泊刀",
+  "item.delightful.silver_knife": "银刀",
+  "item.delightful.brass_knife": "黄铜刀",
+  "item.delightful.invar_knife": "殷钢刀",
+  "item.delightful.constantan_knife": "康铜刀",
+  "item.delightful.electrum_knife": "琥珀金刀",
+  "item.delightful.lead_knife": "铅刀",
+  "item.delightful.nickel_knife": "镍刀",
+  "item.delightful.deorum_knife": "神秘金刀",
+  "item.delightful.reinforced_deorum_knife": "强化神秘金刀",
+  "item.delightful.draco_arcanus_knife": "神秘天龙刀",
+  "item.delightful.fiery_knife": "炽铁刀",
+  "item.delightful.ironwood_knife": "铁木刀",
+  "item.delightful.knightmetal_knife": "骑士金属刀",
+  "item.delightful.steeleaf_knife": "钢叶刀",
+  "item.delightful.living_knife": "活木刀",
+  "block.delightful.pizza_stone": "披萨板",
+  "block.delightful.pizza_peel": "披萨铲",
+  "block.delightful.mini_melon": "迷你西瓜",
+  "block.delightful.acorn_sack": "袋装橡子",
+  "block.delightful.salmonberry_sack": "袋装大树莓",
+  "block.delightful.salmonberry_bush": "大树莓灌木",
+  "farmersdelight.tooltip.prickly_pear_juice": "非常充实的",
+  "farmersdelight.tooltip.azalea_tea": "充实又健康的",
+  "farmersdelight.tooltip.lavender_tea": "非常健康的",
+  "farmersdelight.tooltip.matcha_latte": "健康的",
+  "farmersdelight.tooltip.berry_matcha_latte": "健康的",
+  "farmersdelight.tooltip.ender_nectar": "超自然的",
+  "block.delightful.quartz_cabinet": "石英厨柜",
+  "block.delightful.basalt_cabinet": "玄武岩厨柜",
+  "block.delightful.wild_salmonberries": "野生大树莓",
+  "delightful.matcha.desc": "绿茶粉",
+  "delightful.green_tea_leaf.desc": "从任意树叶上使用刀可能获得。",
+  "delightful.animal_fat.desc": "从一只富含脂肪的动物（猪、牛、鱼……）使用刀可能获得。",

```

```suggestion
  "delightful.animal_fat.desc": "对一只富含脂肪的动物（猪、牛、鱼……）使用刀可能获得。",
```
话说真的按commit suggestion
或者来files里按Add suggestion to batch然后点右上角的绿色按钮就行的

避免出现更改失误的问题